src/elm/App.elm

```diff
@@ -260,6 +269,11 @@ update action model =
       , Effects.none
       )
 
+-- setTimeOut : Int -> Action ->  Effects Action
```

Comment:
@amitaibu 
The compiler fails when I  tried to add the function signature this is why it's commented out.